als-actions/shared/src/test/resources/actions/documentFormatting/json/expected/multiple-syntax-errors.json

```diff
@@ -1,17 +1,18 @@
 {
-  "array" : [
+  "array": [
+
     {
-      "syntax " "error"
+      "syntax ": "error"
     },
+
     {
       "somewhat": "valid",
       "valid": true
     },
     "entry"
   ],
   "paths": {
-    "dd"
-    dd
+    "dd": null
```

Comment:
This line (15) and the previous one with "syntax ": "error" on line 5 are producing different outputs, not sure how to check that from source, are those changes ok?